### PR TITLE
FD-345: Provider - Create Cluster & Download Certificate API bind

### PR DESCRIPTION
**Pull request by BigBadAlien**

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -1,2 +1,2 @@
 REACT_APP_CHAIN_API_BASE=//chain-stage.dccn.ankr.com
```
Comment (starkyru) [inline .env.development]: Does it work with https too?

Comment (BigBadAlien) [reply .env.development]: No, updated

File: src/redux/effects/api/providerSagas.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      });
+
+      if (!hasWallet) {
+        goTo(ProviderRegistrationStepsRoutes.wallet);
+      } else {
+        goTo(ProviderRegistrationStepsRoutes.createCluster);
+      }
+    }
+    // tslint:disable-next-line:no-empty
+  } catch (error) {}
```
Comment (starkyru) [inline src/redux/effects/api/providerSagas.ts]: Add a Sentry call

File: src/screens/Provider/ProviderRegistration/ProviderRegistration.tsx [hunk 3]
```diff
@@ -81,23 +86,60 @@ class ProviderRegistrationComponent extends React.Component<
...
       </PanelWithHeader>
     );
   }
 
+  private handleDownloadCertificates = () => {
+    const { clusterCertificates, clusterDescription } = this.props;
+    const { keysDownloaded } = this.state;
+
+    if (clusterCertificates && !keysDownloaded && clusterDescription) {
+      downloadYamlFile(
```
Comment (starkyru) [inline src/screens/Provider/ProviderRegistration/ProviderRegistration.tsx]: is it failproof? Add try catch!

File: src/utils/files.ts [hunk 4]
```diff
@@ -64,4 +65,16 @@ function downloadCSVFile(data: CSVRecord[][], filename = 'keys.csv') {
   downloadTextFile(exportToCSV(data), filename);
 }
 
-export { downloadCSVFile, exportToCSV, CSVRecord, downloadBinaryFile };
+function downloadYamlFile(data: {}, filename: string) {
+  // TODO process exception
```
Comment (starkyru) [inline src/utils/files.ts]: not todo it, do it!

**Pull request merged**